src/client/sandbox/node/element.js

```diff
@@ -543,6 +546,11 @@ export default class ElementSandbox extends SandboxBase {
         return el.parentNode && domUtils.isShadowUIElement(el.parentNode) || ShadowUI.containsShadowUIClassPostfix(el);
     }
 
+    _isNeedToUpdateUrlResolver (el) {
+        return domUtils.isElementInDocument(el, this.document) &&
```

Comment:
Also add tests for `documentFragment` and for `setProperty(container, 'innerHTML', value) cases